scripts/travis.sh

```diff
@@ -2,7 +2,14 @@
 
 # if this build was triggered as cron job, run e2e tests on iOS
 if [ "${TRAVIS_EVENT_TYPE}" = "cron" ] ; then
-    export SAUCE_PLATFORM_NAME="iOS";
+    DD=$(date +%d)
```

Comment:
Our free sauce labs account can only run limited number of tests everyday.
Hence running android and iOS alternatively on travis cron jobs